Contents/mods/AwayFromZomboid/media/lua/client/AwayFromZomboid/AwayFromZomboid.lua

```diff
@@ -307,7 +312,7 @@ end
 ---@return void
 AwayFromZomboid.manualAFKHook = function(chatMessage, tabId)
     if AwayFromZomboid.getAllowManualAFK() then
-        if chatMessage:getText() == "afk" and chatMessage:getAuthor() == getPlayer():getUsername() then
+        if (chatMessage:getText() == "AFK" or chatMessage:getText() == "afk" or chatMessage:getText() == "Afk") and chatMessage:getAuthor() == getPlayer():getUsername() then
```

Comment:
This could probably be turned into `chatMessage:getText().lower() == "afk"` tbh? I assume/hope PZ's LUA flavour supports that.

Reply:
I think is "string.lower("AFK")" or something like this